gu-provider/README.md

```diff
@@ -1,45 +1,73 @@
 
-# gu-provider
+# Golem Unlimited Provider
 
-Golem Unlimited worker node.
+A worker node for Golem Unlimited. It is subordinate to the [Hub](../gu-hub). 
+It can be run on all three main OS Platforms.
 
-## Install from src
+## Build from src
 
-Clone the repo, go to the `gu-provider` dir and
+To perform build you need working [Rust](https://rustup.rs) on your platform.
+
+### on Ubuntu
+
+Clone the repo, go to the `gu-provider` dir and run:
 ```
-cargo build
+cargo build --release
 ```
-### on MacOS
+### on macOS and Windows
 ```
-cargo run --features "async_docker/ssl gu-hardware/clinfo"
+cargo build --features "async_docker/ssl gu-hardware/clinfo" --release
 ```
-* ssl is needed in order to communicate with docker 
-* clinfo is needed for proper GPU detection
+* `ssl` is needed when you want to utilise docker execution environment within Golem Unlimited  
+* `clinfo` is needed for proper GPU detection
 
-## Command line options
+## Run
 
-Run server
+To run the Provider invoke:
 ```
-$ gu-provider server run
+$ gu-provider --user server run
 ```
 
-Start server as daemon
-```
-$ gu-provider server start
-```
+you can omit `--user` when you have administrative priviledges.
 
-Check server status
-```
-$ gu-provider server status
-```
+Check other commands by invoking:
 
-Stop server daemon
 ```
-$ gu-provider server stop
+$ gu-provider help
 ```
 
-List available lan peers
+## Configuration
+
+### GUI for Linux, macOS and Windows
```

Comment:
Great job @filipgolem!
Can you add also section for manually connecting the Hub with CLI as it is here

https://devcon.golem.network/#/Tutorial?id=_1-join-golem-unlimited-cluster

`gu-provider hubs connect <GU_HUB_ADDR>:61622`
?